content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -110,86 +51,57 @@ follow
+to `kubeadm init`. To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you 
+must specify an IPv6 address, for example `--apiserver-advertise-address=fd00::101`
+
+Now run:
 
 ```bash
-kubeadm init
+kubeadm init <args> 
 ```
 
 **Notes:**
 
-- Please refer to the [kubeadm reference guide](/docs/reference/setup-tools/kubeadm/kubeadm/) if you want to
-read more about the flags `kubeadm init` provides. You can also specify a
-[configuration file](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file) instead of using flags.
-- You need to choose a Pod Network Plugin in the next step. Depending on what
-third-party provider you choose, you might have to set the `--pod-network-cidr` to
-something provider-specific. The tabs below will contain a notice about what flags
-on `kubeadm init` are required.
-- Unless otherwise specified, kubeadm uses the default gateway's network interface
-to advertise the master's IP. If you want to use a different network interface, specify
-`--apiserver-advertise-address=<ip-address>` argument to `kubeadm init`. To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you must specify an IPv6, e.g. `--apiserver-advertise-address=fd00::101`
-- If you would like to customise control plane components including optional IPv6 assignment to liveness probe for control plane components and etcd server, you can do so by providing extra args to each one, as documented [here](/docs/admin/kubeadm#custom-args).
-- `kubeadm init` will first run a series of prechecks to ensure that the machine
-is ready to run Kubernetes. It will expose warnings and exit on errors. It
-will then download and install the cluster database and control plane
-components. This may take several minutes.
-- You can't run `kubeadm init` twice without tearing down the cluster in between
-([unless you're upgrading from v1.6 to v1.7](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)),
-see [Tear Down](#tear-down).
+- For more information about `kubeadm init` arguments, see [the kubeadm reference guide.](/docs/reference/setup-tools/kubeadm/kubeadm/) 
+- For a complete list of configuration options, see [the configuration file documentation.](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file)
+- To customise control plane components, including optional IPv6 assignment to liveness probe for control plane components and etcd server, you provide extra arguments to each component. See  as documented [here](/docs/admin/kubeadm#custom-args).
+- To run `kubeadm init` again, you must first [tear down the cluster](#tear-down).
 - If you join a node with a different architecture to your cluster, you should create a separate
 Deployment or DaemonSet for `kube-proxy` and `kube-dns` on the node. This is because the Docker images for these
 components do not currently support multi-architecture.
 
+`kubeadm init` first runs a series of prechecks to ensure that the machine
```

Comment:
Maybe "It prints warnings if applicable, and if there is an error, it exits with the error message."

Maybe "If there are no errors, kubeadm downloads and installs the components of the cluster plane. This may take several minutes."

And in the prerequisites above, you did not specify that the master needs internet connectivity, but this step implies that it does. Also that you probably need sudo access? But you didn't say to run `kubeadm init` with sudo. Can you clarify?